### DISCUSSION TITLE
What is the best way to update the supergraph

### DISCUSSION THREAD

**Discussion by mohit61** [apollographql/router#1538] [Implementation]
Apollo router uses the supergraph generated by the schema registry. 
What would be the best way to update the supergraph when new version of supergraph is released/updated by schema registry? One of the way is to poll the supergraph from the schema registry and replace the existing supergraph with the new one. What do you think?

**Comment by abernix** [comment]
This is supported natively.  All you need to do is connect the Apollo Router to Studio by following [the instructions in our documentation](https://www.apollographql.com/docs/router/managed-federation/setup#4-connect-the-router-to-studio).  It's as simple as setting your `APOLLO_KEY` and `APOLLO_GRAPH_REF` environment variables.
